core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -102,14 +105,22 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
   val entityType = "test-type"
   val failImportStatusUUID = UUID.randomUUID()
 
-  def testAttributes(importId: UUID) = Map(
+  def testImportServiceAttributes(importId: UUID): Map[String, String] = Map(
     "workspaceName" -> workspaceName.name,
     "workspaceNamespace" -> workspaceName.namespace,
     "userEmail" -> userInfo.userEmail.toString,
     "upsertFile" -> s"$bucketName/${importId.toString}",
     "jobId" -> importId.toString
   )
 
+  def testAttributes(importId: UUID): Map[String, String] = Map(
+    "isCWDS" -> "true",
+    "workspaceId" -> workspaceId.toString,
+    "userEmail" -> userInfo.userEmail.toString,
+    "upsertFile" -> s"$bucketName/${importId.toString}",
+    "jobId" -> importId.toString
+  )
```

Comment:
by rewriting the `testAttributes()` method to produce a cWDS pub/sub message, ALL test coverage in this file is now testing the cWDS code paths instead of Import Service code paths (except the happy-path tests, see comment below)